modules/home/transformations/create-site-settings-url.php

```diff
@@ -0,0 +1,84 @@
+<?php
+namespace Elementor\Modules\Home\Transformations;
+
+use Elementor\Core\Base\Document;
+use Elementor\Plugin;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+class Create_Site_Settings_Url extends Base\Transformations_Abstract {
+
+	const URL_TYPE = 'site_settings';
+
+	public function transform( array $home_screen_data ): array {
+		if ( empty( $home_screen_data['get_started'] ) ) {
+			return $home_screen_data;
+		}
+
+		$site_settings_url_config = $this->get_site_settings_url_config();
+
+		$home_screen_data['get_started'][0]['repeater'] = array_map( function( $repeater_item ) use ( $site_settings_url_config ) {
+			if ( 'Site Settings' !== $repeater_item['title'] ) {
+				return $repeater_item;
+			}
+
+			return array_merge( $repeater_item, $site_settings_url_config );
+		}, $home_screen_data['get_started'][0]['repeater'] );
+
+		return $home_screen_data;
+	}
+
+	private function get_site_settings_url_config() : array {
+		$existing_elementor_page = $this->get_elementor_page();
+		$site_settings_url = ! empty( $existing_elementor_page ) ?
+			$this->get_elementor_edit_url( $existing_elementor_page->ID ) :
+			Plugin::$instance->documents->get_create_new_post_url( 'page' );
+
+		return [
+			'new_page' => empty( $existing_elementor_page ),
+			'url' => $site_settings_url,
```

Comment:
not necessary to escape here: 
- link is generated in WP 
- don't want to mess with unescaping in JS later